crates/cli/src/meta/types/authoring/v2.rs

```diff
@@ -185,6 +200,55 @@ impl AuthoringMetaV2 {
 
         Ok(meta)
     }
+
+    /// checks if the given contract implements ERC165
+    /// the process is done as described per ERC165 specs:
+    /// https://eips.ethereum.org/EIPS/eip-165#how-to-detect-if-a-contract-implements-erc-165
+    pub async fn supports_erc165(client: &ReadableClientHttp, contract_address: Address) -> bool {
```

Comment:
isn't this a generic function that could go elsewhere? it's not specifically related to AuthoringMetaV2 is it?

Reply:
relocated ./src/meta/mod.rs